docs/docs/index.md

```diff
@@ -18,109 +18,86 @@ on Windows, Linux and OSX.
 ## Installation
 
 Poetry provides a custom installer that will install `poetry` isolated
-from the rest of your system by vendorizing its dependencies. This is the
-recommended way of installing `poetry`.
+from the rest of your system.
+
+This is the recommended way of installing `poetry`.
```

Comment:
I would like us to consider not recommending one method but rather treat script, pip and pipx as equals.

Reply:
I removed the recommendation.